homeassistant/components/bmw_connected_drive.py

```diff
@@ -6,7 +6,6 @@
 """
 import logging
 import datetime
```

Comment:
Please sort 🔤 within groups standard library, 3rd party and homeassistant imports.